cmd/nerdctl/run_test.go

```diff
@@ -449,3 +449,31 @@ COPY --from=builder /go/src/logger/logger /
 	assert.Check(t, strings.Contains(log, "foo"))
 	assert.Check(t, strings.Contains(log, "bar"))
 }
+
+func TestNewTerminalDetachedMode(t *testing.T) {
```

Comment:
This test failed on windows.

Also suggest to include `Run` in the func name, something like `TestRunWithTtyAndDetached`.

Reply:
> This test failed on windows. Not sure if it's due to some issue or not supported on windows.
> 

it seems that the error location is different every time

Reply:
> This test failed on windows. Not sure if it's due to some issue or not supported on windows.
> 


using `-d` will use the json-file log driver and cause the windows test  fail